DOCUMENTATION.md

```diff
@@ -239,6 +239,7 @@ _The production flag (`--production` or `-p`) is a shortcut for setting the envi
 - `npm start` starts the development server with hot module reloading, etc.
 - `npm start -- --fast-refresh` (**experimental**) starts the development server with react-refresh, etc.
 - `npm start -- --fast-dev` (**experimental**) starts the development server with some optimizations for faster builds (_warning:_ this will lead to different bundles than in production, use with caution).
+- `npm start -- --experimental-esbuild` (**experimental**) starts the development server with [ESBuild]()https://esbuild.github.io/ instead of [Babel](https://babeljs.io/) (_warning:_ this will lead to different bundles than in production, use with caution). [Read here how to set it up.](packages/webpack#--experimental-esbuild-experimental)
```

Comment:
> this will lead to different bundles than in production, use with caution

That's not quite true as the esbuild mode could also be used in production.

Reply:
Good point!